aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -15,7 +19,13 @@ import { Box } from 'grommet';
 
 ## Guidance
 
-Everything that you need to know about toasts and more.
+Toasts are a low-attention notification mechanism primarily for use in confirming task receipt and completion.
+
+Toasts are intended to be non-intrusive and brief. A user must be able to read and comprehend a toast message in ~8 seconds before the toast automatically closes itself.
+
+Toasts are not a guaranteed delivery mechanism. They can easily be missed and are never shown while the user is focused on performing a task.
```

Comment:
"guaranteed delivery mechanism" sounds like a back-end engineering specification. Can we phrase this to be a bit more user/human centered? Perhaps something like: "The user may not see a toast notification." which connects to the following paragraph a bit more.